decryptor/base/decryptionNotification.go

```diff
@@ -0,0 +1,71 @@
+package base
+
+import "context"
+
+// ColumnInfo interface describe available metadata for column
+type ColumnInfo interface {
+	Index() int
+	Alias() string
+}
+
+type columnInfo struct {
+	index int
+	alias string
+}
```

Comment:
When I added ColumnInfo, I knew that at the future we will be able to provide more info about columns (type, db specific info, some other context)
ClientZoneInfo it's final state of data which will be useful in that context. For now to decrypt info we need client_id or zone_id and ZoneMode.
But when I write these rows, I think that it would be better to use ClientZoneInfo as interface too...